scss/_git_repo.scss

```diff
@@ -1,26 +1,192 @@
+@import "color";
+@import "mixins";
+
 .git-repo-areas {
   margin: auto;
 }
 
 .git-repo {
-  .repo-name {
-    font-size: 1.2rem;
-  }
-  .link-out {
+
+  /* Repo item */
+  .repo-item {
     position: relative;
-    padding: 0.3rem 0.8rem;
-    display: inline-block;
-    vertical-align: top;
-    height: 1rem;
-    &:before {
+    padding: 24px 55px 24px 0;
+    border-bottom: 1px solid $light-grey;
+
+    /* Repo name */
+    & > .name {
```

Comment:
I have put a the targeted css so in case there's another `.name` in the child of another child of `.repo-item`, it won't be affected by this css.